model/RoleBandResponse.ts

```diff
@@ -0,0 +1,6 @@
+export class RoleBandResponse {
```

Comment:
Ideally this would be exported as an interface or as a type

```suggestion
export interface RoleBandResponse
```